spack.yaml

```diff
@@ -25,24 +24,16 @@ spack:
   specs:
     - $ROOT_SPEC
   packages:
-    oasis3-mct:
-      require:
-        - '@git.2023.11.09'
-    netcdf-c:
-      require:
-        - '@4.9.2'
-    netcdf-fortran:
-      require:
-        - '@4.6.1'
+    access-test-component:
+     require:
+        - '@main'
     openmpi:
       require:
         - '@4.1.5'
         - 'cppflags="-diag-disable=10441"'
```

Comment:
```suggestion
```

Reply:
This is not needed anymore. The fix is in our Spack v0.22 branch.

Reply:
I think I left them in as an example, but will remove.